src/sentry/static/sentry/app/stores/teamStore.jsx

```diff
@@ -35,6 +35,18 @@ const TeamStore = Reflux.createStore({
     if (!item) {
       this.items.push(response);
     } else {
+      // Slug was changed
+      // Note: This is the proper way to handle slug changes but unfortunately not all of our
```

Comment:
we're not actually reloading here -- should we?

Reply:
We reload in the form submit success handler